src/content/en/fundamentals/discovery-and-monetization/payment-request/index.md

```diff
@@ -306,13 +325,16 @@ The [`complete`](https://www.w3.org/TR/payment-request/#complete) method tells t
 
 If you are a merchant selling physical goods, you may want to collect the user's shipping address using the Payment Request API. This is accomplished by adding `requestShipping: true` to the `options` parameter. With this parameter set, "Shipping" will be added to the UI, and users can select from a list of stored addresses or add a new shipping address.
 
+You can alternatively use "Delivery" or "Pickup" instead of "Shipping" in the UI by specifying `shippingType`. This is solely display purpose.
```

Comment:
Change 'this is solely display purpose' to 'this is solely for display purposes'.